packages/katello/katello-repos/katello-repos.spec

```diff
@@ -23,17 +32,18 @@ BuildRequires: sed
 Defines yum repositories for Katello and its sub projects, Candlepin and Pulp.
 
 %package -n katello-client-repos
-Summary:  Definition of yum repositories for Katello clients
-Group:    Applications/System
+Summary:   Definition of yum repositories for Katello clients
+Group:     Applications/System
+Conflicts: katello-repos
```

Comment:
This won't build in OBS without this, as katello-repos and katello-client-repos have files that conflict with each other.